docs/roadmap.md

```diff
@@ -22,6 +22,7 @@ Windowing | ❌ Not Supported | ❌ Not Supported | ✅ Supported |
 Deploy Options: |  
 • In-app | ✅ Supported | ✅ Supported | ✅ Supported | 
 • Fx Package | ✅ Supported | ✅ Supported | ✅ Supported | 
+[Push Notifications](https://github.com/microsoft/ProjectReunion/blob/main/specs/PushNotifications/PushNotifications-spec.md) | ❌ Not Supported | ❌ Not Supported | 🔄 Preview |
```

Comment:
Can you move this above Deploy Options and below Windowing? And same with the unpackaged apps table below too

![image](https://user-images.githubusercontent.com/13246069/122299535-faabdd80-ceb2-11eb-8704-6bd9520ef7eb.png)